src/lib.rs

```diff
@@ -29,6 +29,10 @@ mod jdbc;
 #[macro_use]
 mod utils;
 
+#[cfg(feature = "wasm")]
```

Comment:
(linked to the cargo.toml comment): would it be possible to `cfg(target = "wasm32-unknown-unknown")]` here?

Reply:
https://github.com/prisma/connection-string/pull/6/commits/866e443c88015c023ff471e5c7c93271ba97f973